code/game/objects/items/tools/powertools.dm

```diff
@@ -0,0 +1,153 @@
+/obj/item/powertool
+	name = "Power tool"
+	desc = "A basic powertool that does nothing."
+	icon = 'icons/obj/tools.dmi'
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	w_class = WEIGHT_CLASS_SMALL
+	materials = list(/datum/material/iron=150,/datum/material/silver=50,/datum/material/titanium=25)
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30)
+	flags_1 = CONDUCT_1
+	slot_flags = ITEM_SLOT_BELT
+	toolspeed = 0.7
+
+/obj/item/powertool/attack_self(mob/user)
+	toggle_mode(user)
+
+/obj/item/powertool/proc/toggle_mode(mob/user)
+	return
+
+//Hand Drill
+
+/obj/item/powertool/hand_drill
+	name = "hand drill"
+	desc = "A simple powered hand drill. It's fitted with a screw bit."
+	icon_state = "drill_screw"
+	item_state = "drill"
+
+	force = 8 //might or might not be too high, subject to change
+	throwforce = 8
+	throw_speed = 2
+	throw_range = 3//it's heavier than a screw driver/wrench, so it does more damage, but can't be thrown as far
+
+	hitsound = 'sound/items/drill_hit.ogg'
+
+	tool_behaviour = TOOL_SCREWDRIVER
+	usesound = 'sound/items/drill_use.ogg'
+
+/obj/item/powertool/hand_drill/toggle_mode(mob/user)
+	playsound(get_turf(user), 'sound/items/change_drill.ogg', 50, 1)
+	if(tool_behaviour == TOOL_SCREWDRIVER)
+		to_chat(user, "<span class='notice'>You attach the bolt driver bit to [src].</span>")
+		become_wrench()
+	else
+		to_chat(user, "<span class='notice'>You attach the screw driver bit to [src].</span>")
+		become_screwdriver()
+
+/obj/item/powertool/hand_drill/proc/become_wrench()
+	icon_state = "drill_bolt"
+	tool_behaviour = TOOL_WRENCH
+
+	hitsound = null
+
+	attack_verb = list("drilled", "screwed", "jabbed")
+	throw_range = 7
+
+/obj/item/powertool/hand_drill/proc/become_screwdriver()
+	icon_state = "drill_screw"
+	tool_behaviour = TOOL_SCREWDRIVER
+
+	hitsound = 'sound/items/drill_hit.ogg'
+
+	attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked")
```

Comment:
I think these attack_verb 's should be swapped

Reply:
Good spot